src/test/java/org/apache/commons/lang3/ArchUtilsTest.java

```diff
@@ -40,6 +40,8 @@ public class ArchUtilsTest extends AbstractLangTest {
     private static final String X86 = "x86";
     private static final String X86_64 = "x86_64";
     private static final String AARCH_64 = "aarch64";
+    private static final String RISCV64 = "riscv64";
```

Comment:
@kxxt 
Thank you for the update.
Follow the style in this class: "RISCNUM" -> "RISC_NUM".

Reply:
@garydgregory  Thanks for your quick review. It is fixed now.